### PR TITLE
Updated New user in about page

### PR DESCRIPTION
**Pull request by Lakshmipriya0812**

### CODE REVIEW

File: client/public/about.html [hunk 1]
```diff
@@ -501,6 +501,17 @@ <h5 class="fw-bold">Camron Olsen</h5>
                 <p class="text-muted">Drexel Fall 2024</p>
               </div>
             </div>
+          </div> <div class="col-md-4">
```
Comment (CamronOlsen) [inline client/public/about.html:504]: Please move the new div to the next line to maintain established formatting.

File: client/public/about.html [hunk 2]
```diff
@@ -501,6 +501,17 @@ <h5 class="fw-bold">Camron Olsen</h5>
...
+            <div class="card text-center mb-3 box-shadow-custom">
+              <div class="card-body p-0 pb-4">
+                <div class="mb-4">
+                  <img class="img-fluid" src="/static/img/man-user-circle-icon.svg" alt="No Profile Picture">
+                </div>
+                <h5 class="fw-bold">Lakshmipriya Rajesh Kanna</h5>
+                <div class="text-muted">MS in Software Engineering</div>
+                <p class="text-muted">Drexel University - Expected Graduation(Winter 2025)</p>
```
Comment (CamronOlsen) [inline client/public/about.html:512]: Please change this to "Drexel Winter 2025" to maintain syntax of all other profiles.

**Comment by CamronOlsen** [comment]
@Lakshmipriya0812 There are a few things that I would like you to take a look at and resolve. Not trying to be nit picky, just trying to maintain the formatting of things.

File: client/public/about.html (hunk 1 above)
Comment (Lakshmipriya0812) [reply client/public/about.html:504]: Yeah sure

File: client/public/about.html [hunk 3]
```diff
@@ -501,6 +501,17 @@ <h5 class="fw-bold">Camron Olsen</h5>
                 <p class="text-muted">Drexel Fall 2024</p>
               </div>
             </div>
+          </div> <div class="col-md-4">
+            <div class="card text-center mb-3 box-shadow-custom">
+              <div class="card-body p-0 pb-4">
+                <div class="mb-4">
+                  <img class="img-fluid" src="/static/img/man-user-circle-icon.svg" alt="No Profile Picture">
```
[the remark this answers is not in the document]
Comment (Lakshmipriya0812) [reply client/public/about.html:508]: I did women but not sure, Will sure change it and update it again

**Pull request merged**